### PR TITLE
process and freeze all patients

### PR DESCRIPTION
**Pull request by sharleenawinja**
# Process and freeze all patients

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/RdeSync/Moh731Sync.component.tsx [hunk 1]
```diff
@@ -179,6 +168,21 @@ const Moh731SyncQueueComponent = () => {
...
+    reportingMonth: string
+  ) => {
+    const payload = {
+      userId: user?.uuid,
+      reportingMonth: reportingMonth,
+      patientIds: [personId],
+    };
+    const result = await processQueuedPatients(payload);
+    await fetchMoh731SyncQueue(selectedMonth).then(setPatients);
+    console.log(result);
```
Comment (kipchubett) [inline src/components/RdeSync/Moh731Sync.component.tsx]:
```suggestion
```
remove console log

File: src/components/RdeSync/Moh731Sync.component.tsx [hunk 2]
```diff
@@ -215,14 +218,43 @@ const Moh731SyncQueueComponent = () => {
...
+  const handleProcessAll = async () => {
+    const payload = {
+      userId: user?.uuid,
+      reportingMonth: selectedMonth,
+      patientIds: patientIds,
+    };
+
+    const result = processQueuedPatients(payload);
+    await fetchMoh731SyncQueue(selectedMonth).then(setPatients);
+    console.log(result);
```
Comment (kipchubett) [inline src/components/RdeSync/Moh731Sync.component.tsx]:
```suggestion
``
Ditto

**Review by kipchubett — CHANGES_REQUESTED** [review]
Remove superfluous  `console logs`
Otherwise LGTM!

**Comment by kipchubett** [comment]
Also resolve the conflicts

File: src/components/RdeSync/Moh731Sync.component.tsx (hunk 2 above)
Comment (sharleenawinja) [reply src/components/RdeSync/Moh731Sync.component.tsx]: is there a way i can remove line 231 and 232 right here in this PR? I added them by mistake

Comment (kipchubett) [reply src/components/RdeSync/Moh731Sync.component.tsx]: Yes by committing the suggestion

**Pull request closed without merging**